components/src/Tables/Table.js

```diff
@@ -1,38 +1,98 @@
 import React from 'react';
 import styled from 'styled-components';
+import { colour, space, font, radius, shadow } from '@nulogy/tokens';
+import QuietButton from '../Button/QuietButton';
 
-const borderStyle = ({ theme }) => `1px solid ${theme.colour.neutral[400]}`;
+const borderWidth = 1;
+
+const withoutBorder = value => value - borderWidth;
+
+const lineHeight = name => font.lineHeight[name] * font.size.medium;
```

Comment:
This is now used in two places ... it may be a good candidate for extraction ... perhaps to the Text component?